Tsw 53 pr this

CarbonBloem opened the pull request.

Commit:
Majority of evm to native transfers work

Commit:
Loading transactions

Commit:
Date padding fix

Commit:
Updated contract abi

Commit:
Progress so far

CarbonBloem committed:
Antelope bridge added

CarbonBloem committed:
Merge remote-tracking branch 'origin/dev' into TSW-53-add-bsc-ethereum-to-interface

CarbonBloem committed:
- Removed package-lock.json
- Removed UalDialog2.vue
- Can add memo now
- Better network selection for the from selector.

CarbonBloem committed:
Added telosd contract interaction

CarbonBloem committed:
Merge branch 'dev' into TSW-53-PR-THIS

CarbonBloem committed:
lock file changed and working

JPBeukes reviewed (changes requested):
- Sending from EOS to Telos gives a transaction id link to teloscan, not bloks.io. I'm referring to this popup:
![image](https://user-images.githubusercontent.com/30291405/203624684-3055e461-a12c-423c-b8bc-901dc961de63.png)
- Auto login (using local storage) has been disabled.
- Metamask does not prompt you to select a different network when you switch evm networks in the app.
- Sending tokens from Telos native to any of the EVM chains does not work. Might be a reporter issue, but sending from one of the EVM chains to Telos native works, so it might not.

CarbonBloem committed:
I do believe this is all that is required for this ticket, but blocked by reporter issue for testing.

CarbonBloem committed:
Switch metamask network done

CarbonBloem committed:
Merge pull request #37 from Yknot-BCS/TSW-135-metamask-network-switch

Switch metamask network done

CarbonBloem committed:
In this commit:

- re-enabled the local storage for auto login.
- fixed the sending max bug not proceeding to next step.
- send tx dialog links fixed

CarbonBloem committed:
needed to be part of previous commit

CarbonBloem committed:
Last bugs fixed

CarbonBloem committed:
Merge pull request #38 from Yknot-BCS/TSW-137-reporters-bug

Tsw 137 reporters bug

Dewald928 committed:
chore: put back wax cloud wallet

Dewald928 committed:
chore: Better avatar handling

Dewald928 commented on src/pages/Bridge.vue:
```diff
@@ -28,20 +34,21 @@
 export default {
   components: {
     bridgeStepper,
+    bridgeStepper2,
```
I assume bridge 1 is evm and stepper 2 is antelope? bit unclear

CarbonBloem replied:
Yes, that is true. It is used for the antelope bridge.

Dewald928 commented on src/store/account/actions.js:
```diff
@@ -4,29 +4,34 @@ export const login = async function (
   try {
     commit("setLoadingWallet", authenticator.getStyle().text);
     await authenticator.init();
+    console.log(account);
```
logs

CarbonBloem replied:
Will remove all the console logs in the project.

Dewald928 reviewed (changes requested):
Just a few questions and tweaks

CarbonBloem commented on src/components/bridge/ConnectTo.vue:
```diff
@@ -0,0 +1,289 @@
+<template>
```
It is used for the antelope section (bridgeStepper2)

CarbonBloem commented on src/components/bridge/Connect2.vue:
```diff
@@ -0,0 +1,298 @@
+<template>
```
It is used for the antelope section (bridgeStepper2)

CarbonBloem committed:
Console.log cleanup:
- Commented out console.logs
- Removed previously commented out console.logs.
- Left console.logs in that gives an error indication.

Dewald928 reviewed (approved):
All seems well.
